### PR TITLE
[WIP]Enable custom op and avoid graph breaks

### PR DESCRIPTION
**Pull request by ZhangLirong-amd**

### CODE REVIEW

**Comment by ZhangLirong-amd** [comment]
problem in accuracy:

```
add
mul
sub
div
get_graph_buffer_ipc_meta
get_meta_buffer_ipc_handle
aiter_sigmoid
aiter_tanh
pa_fwd_naive
mha_batch_prefill
mha_bwd
mha_fwd
mha_varlen_bwd
mha_varlen_fwd
all_reduce_asm
all_reduce_rmsnorm
all_reduce_rmsnorm_quant
fmha_v3_bwd
fmha_v3_fwd
fmha_v3_varlen_bwd
pa_fwd
RocSolIdxBlas
```

File: aiter/ops/gemm_op_a8w8.py [hunk 1]
```diff
@@ -13,6 +13,9 @@
 )
 from ..utility import dtypes
 from ..jit.utils.chip_info import get_cu_num
+from torch.library import Library
+
+aiter_lib = Library("aiter", "FRAGMENT")
```
Comment (valarLip) [inline aiter/ops/gemm_op_a8w8.py:18]: need clean?

File: aiter/ops/moe_op.py [hunk 2]
```diff
@@ -180,15 +180,48 @@ def moe_stage1_g1u1(
     kernelName: str,
     block_m: int,
     ksplit: int = 0,
-    activation: ActivationType = ActivationType.Silu,
-    quant_type: QuantType = QuantType.No,
+    activation: Optional[Enum] = ActivationType.Silu.value,
+    quant_type: int = int(QuantType.No),
```
Comment (valarLip) [inline aiter/ops/moe_op.py]: quant_type: Enum= QuantType.No.value,

File: csrc/include/rocm_ops.hpp [hunk 3]
```diff
@@ -434,6 +434,10 @@
 #define MHA_VARLEN_BWD_ASM_PYBIND                 \
     m.def("fmha_v3_varlen_bwd",                   \
           &aiter::torch_itfs::fmha_v3_varlen_bwd, \
+          py::arg("dp"),                          \
```
Comment (valarLip) [inline csrc/include/rocm_ops.hpp]: revert back?

File: gradlib/gradlib/GemmTuner.py [hunk 4]
```diff
@@ -57,7 +57,9 @@ def call_hipb_mm(input, weight, bias, scale_a, scale_b, solidx, out_dtype):
 
 def call_rocb_mm(inp, w, solidx):
     init_rocblas()
-    return aiter.rocb_mm(inp, w, solidx)
+    out = torch.tensor([])
```
Comment (valarLip) [inline gradlib/gradlib/GemmTuner.py]: wrong shape?

File: aiter/ops/gemm_op_a8w8.py (hunk 1 above)
Comment (ZhangLirong-amd) [reply aiter/ops/gemm_op_a8w8.py:18]: aiter::get_CKGEMM_config_ custom op was registered below to avoid GEMM config break

File: gradlib/gradlib/GemmTuner.py (hunk 4 above)
Comment (ZhangLirong-amd) [reply gradlib/gradlib/GemmTuner.py]: fix it. forget implement fake tensor for rocb_mm

**Review by valarLip — APPROVED** [review]
